### ISSUE TITLE
Dialogs transparent on some wayland compositors

### ISSUE THREAD

**Issue by Readf0x** [catppuccin/gtk#130]
**Describe the bug**
Dialogs have no background color

**Mention your distribution and your theme version**
Running latest hyprland git build form the AUR on Arch linux

**Select how you installed the theme**
- [ ] GitHub Releases
- [x] AUR package 
- [ ] Nixpkg
- [ ] Using the python install script 
- [ ] Flatpak

**Symlinked the gtk-4.0 folder?**
Have you symlinked the `gtk-4.0` folder as mentioned in the [README](https://github.com/catppuccin/gtk/blob/e907775723eb68a5628c1bcd718b9d822cea9aee/README.md?plain=1#L85-L93)? **This is not applicable for users who have installed the theme with the Nixpkg.**
yes

**Expected behavior**
Background should not be transparent

**Screenshots**
![image](https://github.com/catppuccin/gtk/assets/57838054/4d11a025-bc82-4680-988d-1ddbd3f46521)

**Additional context**
This is not a libadwaita issue, happens on all dialogs that use the gtk portal

**Comment by rubyowo** [comment]
Duplicate of #128

**Comment by sgoudham** [comment]
Looks like we're waiting on https://github.com/vinceliuice/Colloid-gtk-theme/pull/135 being merged upstream.

**Comment by ForgotMyPasswd** [comment]
This was merged, will it be implemented into this theme? I'm still gettng this issue on certain dialog boxes for Xournal++ and other apps on Hyprland-git.

**Comment by sgoudham** [comment]
Hey @ForgotMyPasswd, are you getting this issue on the latest release: https://github.com/catppuccin/gtk/releases/tag/v0.7.4 release ?

**Comment by ForgotMyPasswd** [comment]
The issue has been fixed in the 0.7.4 release
